ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The license in the original project being merged here has been removed with my express written approval.